specs/v1.yaml

```diff
@@ -422,6 +422,441 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks:
+    get:
+      summary: List Webhooks
+      description: List Webhooks for a subscription
+      operationId: list-webhooks
+      security:
+        - auth0:
+          - "read:webhooks"
+      parameters:
+        - name: pageLimit
+          in: query
+          description: the number of results to return; defaults to 10, max is 100
```

Comment:
are we actually enforcing these limits for this endpoint?

Reply:
apparently all of our list methods enforce this limit... See `RequestUtils.getPageLimit`